kvdb-rocksdb/src/lib.rs

```diff
@@ -229,19 +231,48 @@ struct DBAndColumns {
 	column_names: Vec<String>,
 }
 
+fn static_property_or_warn(db: &DB, prop: &str) -> usize {
+	match db.property_value(prop) {
```

Comment:
Maybe?
```suggestion
	match db.property_int_value(prop) {
```
…returns `Result<Option<u64>, Error>`

Reply:
I have not checked but I'm not sure that this will get the aggregated number (i.e. for all column families). I think it's worth checking so we don't end up counting memory for the default column only. See https://github.com/facebook/rocksdb/blob/master/include/rocksdb/db.h#L870

Reply:
There is a different method to get property value for specific column familiy, `property_value_cf`, so I expect this one I use should work for all column families, I'll try to add a test

Reply:
@dvdplm Added